var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -86,7 +86,7 @@ class Openblas(MakefilePackage):
 
     # Fixes compilation error on POWER8 with GCC 7
     # https://github.com/xianyi/OpenBLAS/pull/1098
-    patch('power8.patch', when='@0.2.18:0.2.19 %gcc@7.1.0: target=ppc64')
+    patch('power8.patch', when='@0.2.18:0.2.19 %gcc@7.1.0: target=power8')
```

Comment:
Does the patch makd sense for power9 too?  Ah, ok, I looked at it -- the files this modifies are definitely power8-specific.